nmi_tools/glue/build/build_uw_deb.sh

```diff
@@ -83,13 +83,10 @@ if [ "$VERSION_CODENAME" = 'bullseye' ]; then
     true
 elif [ "$VERSION_CODENAME" = 'bookworm' ]; then
     dch --distribution $dist --nmu 'place holder entry'
-elif [ "$VERSION_CODENAME" = 'bionic' ]; then
-    true
 elif [ "$VERSION_CODENAME" = 'focal' ]; then
-    dch --distribution $dist --nmu 'place holder entry'
+    true
```

Comment:
This changes `focal` from `dch --distribution $dist --nmu 'place holder entry'` to `true`.  Can't tell if that's intentional or not.

Reply:
Explained in-person.  Thanks.